webapp/app/assessment/fra/sections/originalDataPoint/actions/updateSectionTables.js

```diff
@@ -88,3 +87,42 @@ export const getUpdateTablesWithOdp = (state, odp, draft = true) => {
   actions.push(getUpdateForestCharacteristics(state, odp, draft))
   return actions.filter((action) => !!action)
 }
+
+// ====== No Original Data Point
+const getFraNoNDPByYear = (year, fraNoNDPs) => R.find(R.propEq('year', year))(fraNoNDPs)
+
+const extentOfForestOdpToNoOdp = (state, year) => {
+  const section = FRA.sections['1'].children.a
+  const sectionName = section.name
+  const tableName = section.tables.extentOfForest
+  const assessmentType = FRA.type
+  const fraNoNDPs = AssessmentState.getFraNoNDPs(assessmentType, sectionName, tableName)(state)
+  if (!fraNoNDPs) {
+    return null
+  }
+  const datumOdp = getFraNoNDPByYear(year, fraNoNDPs)
+  return getUpdateSectionTable(state, sectionName, tableName, datumOdp)
+}
+
+const forestCharacteristicsOdpToNoOdp = (state, year) => {
+  const section = FRA.sections['1'].children.b
+  const sectionName = section.name
+  const tableName = section.tables.forestCharacteristics
+  const assessmentType = FRA.type
+  const fraNoNDPs = AssessmentState.getFraNoNDPs(assessmentType, sectionName, tableName)(state)
+  if (!fraNoNDPs) {
+    return null
+  }
+  const datumOdp = getFraNoNDPByYear(year, fraNoNDPs)
+  return getUpdateSectionTable(state, sectionName, tableName, datumOdp)
+}
+
+export const getUpdateTablesWithNotOdp = (state, year) => {
+  if (!FRA.years.includes(year)) {
+    return null
+  }
+  const actions = []
+  actions.push(extentOfForestOdpToNoOdp(state, year))
+  actions.push(forestCharacteristicsOdpToNoOdp(state, year))
+  return actions.filter((action) => !!action)
+}
```

Comment:
the lines above can become:
```
const extentOfForest = FRA.sections['1'].children.a
const extentOfForestSectionName = extentOfForest.name
const extentOfForestTableName = extentOfForest.tables.extentOfForest

const forestCharacteristics = FRA.sections['1'].children.b
const forestCharacteristicsSectionName = forestCharacteristics.name
const forestCharacteristicsTableName = forestCharacteristics.tables.extentOfForest

const _getUpdateTablesWithNotOdp = (state, year, sectionName, tableName) => {
  const fraNoNDPs = AssessmentState.getFraNoNDPs(FRA.type, sectionName, tableName)(state)
  if (!fraNoNDPs) {
    return null
  }
  const datumOdp = R.find(R.propEq('year', year))(fraNoNDPs)
  return getUpdateSectionTable(state, sectionName, tableName, datumOdp)
}

export const getUpdateTablesWithNotOdp = (state, year) => {
  if (!FRA.years.includes(year)) {
    return null
  }
  const actions = []
  actions.push(_getUpdateTablesWithNotOdp(state, year, extentOfForestSectionName, extentOfForestTableName))
  actions.push(_getUpdateTablesWithNotOdp(state, year, forestCharacteristicsSectionName, forestCharacteristicsTableName))
  return actions.filter((action) => !!action)
}
```
and use the section names where needed in the file